docs/kind-3000-key-attestation.md

```diff
@@ -0,0 +1,153 @@
+
+This specification incorporates [NIP-73](https://github.com/nostr-protocol/nips/blob/master/73.md) for discoverability of events associated with the external identifier.
+
+## Content Format
+
+The `content` field **MUST** be empty.
+
+### Schema
+
+```json
+"content": ""
+```
+
+## Event Structure
+
+```json
+{
+  "id": "<32-bytes lowercase hex-encoded sha256 of the serialized event data>",
+  "pubkey": "<32-bytes lowercase hex-encoded public key of the event creator>",
+  "created_at": <unix_timestamp>,
+  "kind": 3000,
+  "tags": [
+    ["identifier", "<optional_entity_type>", "<unique_identifier>"],
+    ["network", "<network_name>"],
+    ["scheme", "<signature_scheme>"],
+    ["sequence", "<sequence_number_string>"],
+    ["signature", "<signature_from_external_key>"],
+    ["i", "<network>[:<chain_id>]:address:<external_public_key>"],
+    ["k", "<network>:address"],
+    ["alt", "Key Attestation for <identifier> on <network>"]
+  ],
+  "content": "",
+  "sig": "<64-bytes lowercase hex of the signature of the sha256 hash of the serialized event data, which is the same as the 'id' field>"
+}
+```
+
+## Tags
+
+| Tag Name | Description | Example Format | Required |
+|----------|-------------|----------------|----------|
+| `identifier` | Unique identifier, optionally with entity type | `["identifier", "PillarOne"]` or `["identifier", "pillar", "PillarOne"]` | Yes |
+| `network` | Network or blockchain name | `["network", "zenon"]` | Yes |
+| `scheme` | Cryptographic signature scheme | `["scheme", "ed25519"]` | Yes |
+| `sequence` | Version number for updates | `["sequence", "1"]` | Yes |
+| `signature` | Signature from external key | `["signature", "ea00f1ab2ad34b60080e77b431d806e9ead4a4eee36e41cf58a05859b080a03a4dc8a8b21b4952dacf5f7f97262ffdcde47132a269ca97c69260543cd0dc8604"]` | Yes |
+| `i` | NIP-73 external identifier | `["i", "zenon:1:address:z1qqjnwjjpnue8xmmpanz6csze6tcmtzzdtfsww7"]` | Yes |
+| `k` | NIP-73 identifier type | `["k", "zenon:address"]` | Recommended |
+| `alt` | Human-readable event description | `["alt", "Key Attestation for PillarOne on Zenon Network"]` | Recommended |
+
+### `identifier`
+
+A unique identifier for the entity being attested. The tag has the format `["identifier", "<optional_entity_type>", "<unique_identifier>"]` where the entity type can be omitted when not needed.
+
+The entity type provides semantic clarity by explicitly categorizing the type of entity (e.g., "organization", "username", "pillar"), while omitting it provides flexibility for cases where entity type categorization is not needed.
+
+Examples:
+- `["identifier", "PillarOne"]` - No entity type
+- `["identifier", "pillar", "PillarOne"]` - With entity type
+
+### `network`
+
+The name of the network or blockchain where the external key originates. This provides context for which verification rules to apply.
+
+Example: `["network", "zenon"]`, `["network", "bitcoin"]`
```

Comment:
are keys always associated with just one network?

Reply:
There's no network tag anymore in favor of the `k` tag which already includes that information.

To answer your question, Yes. A single `k` value denotes exactly one network namespace/kind. In Nostr, you could include as many values as you want in a tag, but only the second element (the first value) of single-letter tags will be indexed and recognized by relays.